ansible/roles_ocp_workloads/ocp4_workload_quarkus_super_heroes_demo/tasks/deploy_cicd.yaml

```diff
@@ -0,0 +1,270 @@
+---
+- name: Retrieve Gitea instance
+  kubernetes.core.k8s_info:
+    api_version: gpte.opentlc.com/v1
+    kind: Gitea
+    name: "{{ ocp4_workload_gitea_operator_name }}"
+    namespace: "{{ ocp4_workload_gitea_operator_project }}"
+  register: r_gitea
+
+- name: "[{{ t_project_name_argocd }}] - create project"
+  kubernetes.core.k8s:
+    state: present
+    api_version: project.openshift.io/v1
+    kind: Project
+    name: "{{ t_project_name_argocd }}"
+    definition:
+      metadata:
+        name: "{{ t_project_name_argocd }}"
+        annotations:
+          openshift.io/description: "Quarkus Superheroes ArgoCD"
+          openshift.io/display-name: "Quarkus Superheroes ArgoCD"
+
+- name: "[{{ t_project_name_argocd }}] - create argocd instance"
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ lookup('template', 'argocd/argocd.yaml.j2') }}"
+
+- name: "[{{ t_project_name_argocd }}] - create argocd repo secret"
+  vars:
+    # yamllint disable-line rule:line-length
+    _gitea_repo_url: "{{ r_gitea.resources[0].status.giteaRoute }}/{{ ocp4_workload_quarkus_super_heroes_demo_clone_repo_gitea_user }}/{{ ocp4_workload_quarkus_super_heroes_demo_clone_repo_gitea_repo_name }}-deploy"
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ lookup('template', 'argocd/argocd-repo-secret.yaml.j2') }}"
+
+- name: "[{{ t_project_name_test }}] - create project"
+  kubernetes.core.k8s:
+    state: present
+    api_version: project.openshift.io/v1
+    kind: Project
+    name: "{{ t_project_name_test }}"
+    definition:
+      metadata:
+        annotations:
+          openshift.io/description: "Quarkus Superheroes Test Environment"
+          openshift.io/display-name: "Quarkus Superheroes Test Environment"
+
+- name: "[{{ t_project_name_test }}] - label argocd.argoproj.io/managed-by"
+  kubernetes.core.k8s:
+    state: patched
```

Comment:
Is this even necessary? You just created the namespace. Why not add the label to the namespace creation?

Reply:
Perfect. I had issues adding it al together when creating **Project**, now it works because I am creating a **Namespace**. Thanks.